test/integration/len_test.lua

```diff
@@ -62,3 +62,19 @@ pgroup:add('test_len_empty_space', function(g)
     t.assert_equals(err, nil)
     t.assert_equals(result, 0)
 end)
+
+pgroup:add('test_opts_not_damaged', function(g)
+    local len_opts = {timeout = 1}
+    local new_len_opts, err = g.cluster.main_server.net_box:eval([[
```

Comment:
After luatest update we could write `main_server:eval` without net_box (https://github.com/tarantool/luatest/commit/5d424209b42ad56bd15840f3d826567f10df0ffe)